frontend/app/src/components/asset-manager/managed/ManagedAssetForm.vue

```diff
@@ -120,14 +126,12 @@ const clearFieldErrors = (fields: Array<keyof SupportedAsset>) => {
   fields.forEach(clearFieldError);
 };
 
-watch(address, async () => {
-  const sanitizedAddress = sanitizeAddress(get(address));
-  if (get(address) !== sanitizedAddress) {
-    set(address, sanitizedAddress);
+watch([address, evmChain], async ([addressVal, evmChain]) => {
```

Comment:
```suggestion
watch([address, evmChain], async ([address, evmChain]) => {
```
Since we don't use the address property can we change it here and simplify the fetchTokenDetails too?